hashes/src/sha256t.rs

```diff
@@ -220,15 +220,6 @@ macro_rules! sha256t_tag {
 /// BIP-341. Alternatively, you can supply `hash_bytes` to hash raw bytes. If you have the midstate
 /// already pre-computed and prefer **compiler** performance to readability you may use
 /// `raw(MIDSTATE_BYTES, HASHED_BYTES_LENGTH)` instead.
-///
-/// Both visibility modifiers and attributes are optional and passed to inner structs (excluding
-/// `#[hash_newtype(...)]`). The attributes suffer same compiler performance limitations as in
-/// [`hash_newtype`] macro.
```

Comment:
The performance issue is still *here* it just wasn't in the `sha256t_tag` macro.

What causes performance issues for large inputs (quadratic algo) is our use of `tt` muncher which we still do use in `hash_newtype!` to process the atrributes.

Reply:
Oh, thanks for explaining.